lib/translator.js

```diff
@@ -1,22 +1,89 @@
-// def: wrapper for translators
 'use strict';
 
+// def: wrapper for translators
+
+// TODO: default value?
+
 require('dotenv').config({silent: true});
 
-const Translator = require('mstranslator');
-const client = new Translator({
-  client_id: process.env.TRANSLATE_CLIENT_ID,
-  client_secret: process.env.TRANSLATE_CLIENT_SECRET
-}, true); // true: auto-generate token
+const app = require('../server/server'),
+  Translator = require('mstranslator'),
+  _ = require('lodash'),
+  debug = require('debug')('translator');
 
+// enums
+const CACHE_TYPE = {
+  detect: 'd',
+  translate: 't'
+};
+
+// lazy instantiation
+let client, TranslationCache;
 
 module.exports = {
-  translateToEnglish: translateToEnglish,
-  translate: client.translate.bind(client),
-  detect: client.detect.bind(client)
+  translate(obj={text, to: 'en'}) {
```

Comment:
@justinlueders uses default values, so we can more clearly describe object args, better than we have previously.